lib/datastore/request.js

```diff
@@ -258,13 +258,13 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  *   key: key,
  *   data: {
  *     name: 'DonutShack', // strings
- *     rating: datastore.int(8), // ints
- *     worth: datastore.double(123456.78), // doubles
+ *     rating: gcloud.datastore.int(8), // ints
+ *     worth: gcloud.datastore.double(123456.78), // doubles
  *     numDonutsServed: 45, // detect number type (int or double)
- *     founded: new Date('Tue May 12 2015 15:30:00 GMT-0400 (EDT)'), // dates
+ *     // founded: new Date('Tue May 12 2015 15:30:00 GMT-0400 (EDT)'), // dates
```

Comment:
Oops these were commented out because I have no idea why it throws when given a date object. I'll need to investigate this.